tests/headers/repr-align.hpp

```diff
@@ -0,0 +1,6 @@
+// bindgen-flags: --rust-target 1.24 -- -std=c++11
+
+struct alignas(8) a {
```

Comment:
Could you add another testcase with `alignas(double)`? And probably do so with 32bit Linux as target, given that's where the original issue is.

Also maybe add testcase for things when `alignas` is on field rather than type?

Reply:
Sure, will add. We don't run tests on linux32 though, so I won't do that, since it wouldn't test anything extra.

Reply:
I think if you add `-m32` should be enough?

Reply:
Added